app/src/main/res/layout-land/activity_home_screen.xml

```diff
@@ -7,31 +7,82 @@
     tools:context=".activities.HomeScreenActivity">
 
     <Button
-        android:id="@+id/edit_profile_btn"
+        android:id="@+id/add_likes_btn"
         android:layout_width="@dimen/std_btn_width"
         android:layout_height="@dimen/std_btn_height"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentEnd="true"
+        android:layout_marginBottom="37dp"
+        android:layout_marginEnd="158dp"
+        android:text="@string/add_more_likes" />
+
+    <ImageView
+        android:id="@+id/iv_user_prof_pic"
+        android:layout_width="@dimen/pic_width"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
         android:layout_alignParentTop="true"
-        android:layout_centerHorizontal="true"
-        android:layout_marginTop="@dimen/buttons_vertical_padding"
-        android:enabled="true"
-        android:paddingHorizontal="@dimen/buttons_vertical_padding"
-        android:text="@string/edit_prof_btn" />
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="23dp"
+        android:src="@drawable/com_facebook_profile_picture_blank_square" />
+
+    <TextView
+        android:id="@+id/tv_user_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignEnd="@+id/add_likes_btn"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="40dp"
```

Comment:
Move to dimens.xml

Reply:
Oops I think I missed the entire landscape file! Thanks!